.github/workflows/e2e.yml

```
@@ -0,0 +1,84 @@
+          git add .
+          git commit -m "test: trigger release for trusted-go-releaser PR #${{ github.event.pull_request.number }}
+
+          This is a test commit to validate the trusted-go-releaser workflow
+          using ref: ${{ github.event.pull_request.head.ref }}"
+
+          git push origin "$BRANCH_NAME"
+
+      - name: Create PR with bump:patch label
+        env:
+          GH_TOKEN: ${{ secrets.TEST_TRUSTED_GO_RELEASER_REPO_GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)